make-file-list.sh

```diff
@@ -108,6 +107,7 @@ pacman_list () {
 		do
 			pactree -u "$arg"
 		done |
+		sed 's/>.*//' |
```

Comment:
What I came up with to parse a dependency in msys2/pacdb is
```python
_DEPENDRE = re.compile(r'([^<>=]+)(?:(<=|>=|<|>|=)(.*))?')
```

That might be more complicated than you want to use here though.

Reply:
Done in git-for-windows/build-extra#596